model/app.model.js

```diff
@@ -19,3 +19,18 @@ exports.fetchArticles = (article_id) => {
       return rows[0];
     });
 };
+
+exports.patchArticle = (article_id, inc_votes) => {
+  if (inc_votes === undefined)
```

Comment:
Would psql throw its own error here? If so, could we handle this in our error handling, rather than the model?